arwen/elrondapi/elrondei.go

```diff
@@ -1259,6 +1268,22 @@ func returnData(context unsafe.Pointer, pointer int32, length int32) {
 	output.Finish(data)
 }
 
+//export myDebug
+func myDebug(context unsafe.Pointer, pointer int32, length int32) {
+	runtime := arwen.GetRuntimeContext(context)
+	//output := arwen.GetOutputContext(context)
+	metering := arwen.GetMeteringContext(context)
+
+	gasToUse := metering.GasSchedule().ElrondAPICost.Finish
+	gasToUse += metering.GasSchedule().BaseOperationCost.PersistPerByte * uint64(length)
+	metering.UseGas(gasToUse)
+
+	data, err := runtime.MemLoad(pointer, length)
+
+	fmt.Println("Pointer:", pointer, " Length: ", length)
```

Comment:
delete this debug function. we do not want to fill the nodes screen with these written data.